addons/hashes/fnc_encodeJSON.sqf

```diff
@@ -77,6 +78,19 @@ switch (typeName _object) do {
         };
     };
 
+    case "HASHMAP": {
+        private _json = ((_object toArray false) apply {
+            _x params ["_key", ["_value", objNull]];
+
+            if !(_key isEqualType "") then {
+                _key = str _key;
+            };
```

Comment:
Why not `str` on STRINGs as well? You kind of lose the information of "false" vs false and "1" vs 1 with this now.